src/node/desktop/NEWS-electron.md

```diff
@@ -1,5 +1,9 @@
 # RStudio "Electron" Release Notes
 
+## Version Number Update:
```

Comment:
Contents of this file will eventually be merged into the NEWS file that tells users/admins what changes are in a release. So, new features, important bug fixes. Not dev stuff.

Reply:
Actually, that's true for the other things you added on previous PRs (just noticed this!).

So please revert those, and put this file back to empty state (just the title for now).

You can add dev stuff to the wiki page: https://github.com/rstudio/rstudio/wiki/Electron-Desktop

If you want, you could also add a src/node/desktop/README.md and put a link in it to the wiki page, to make it easier to find.